hr_utilization_analysis/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_my_wizard_user,my.wizard.user,model_hr_utilization_analysis_wizard,hr.group_hr_manager,1,1,1,1
```

Comment:
I'm not sure group is correct. Menu item is located under `hr_timesheet.menu_timesheets_reports`, and in v14 it has `hr_timesheet.group_timesheet_manager` group

Reply:
Yes to view the menu user must have Timesheet manager as well as HR manger rights.

Reply:
I don't follow you. Parent menu is accessible to `hr_timesheet.group_timesheet_manager`, why then this wizard requires `hr.group_hr_manager`?